web/template/add-user.gotmpl

```diff
@@ -0,0 +1,167 @@
+        </div>
+
+        <div class="govuk-form-group {{ if .Errors.firstname }}govuk-form-group--error{{ end }}">
+          <label class="govuk-label" for="f-firstname">First name</label>
+          {{ range .Errors.firstname }}
+            <span class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span> {{ . }}
+            </span>
+          {{ end }}
+          <input class="govuk-input govuk-!-width-two-thirds" id="f-firstname" name="firstname" type="text" autocomplete="name" spellcheck="false">
+        </div>
+
+        <div class="govuk-form-group {{ if .Errors.surname }}govuk-form-group--error{{ end }}">
+          <label class="govuk-label" for="f-surname">Last name</label>
+          {{ range .Errors.surname }}
+            <span class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span> {{ . }}
+            </span>
+          {{ end }}
+          <input class="govuk-input govuk-!-width-two-thirds" id="f-surname" name="surname" type="text" autocomplete="name" spellcheck="false">
+        </div>
+
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">Organisation</legend>
+            <div class="govuk-radios govuk-radios--inline">
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="organisation-1" name="organisation" type="radio" value="COP User">
+                <label class="govuk-label govuk-radios__label" for="organisation-1">
+                  COP User
+                </label>
+              </div>
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="organisation-2" name="organisation" type="radio" value="OPG User" checked>
+                <label class="govuk-label govuk-radios__label" for="organisation-2">
+                  OPG User
+                </label>
+              </div>
+            </div>
+          </fieldset>
+        </div>
+        
+        <div class="govuk-form-group {{ if .Errors.roles }}govuk-form-group--error{{ end }}">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">Roles</legend>
+            {{ range .Errors.roles }}
+              <span class="govuk-error-message">
+                <span class="govuk-visually-hidden">Error:</span> {{ . }}
+              </span>
+            {{ end }}
+            
+            <div class="govuk-checkboxes govuk-checkboxes--small">
+              <div class="govuk-checkboxes__item">
+                <input class="govuk-checkboxes__input" id="f-roles" name="roles" type="checkbox" value="Case Manager">
+                <label class="govuk-label govuk-checkboxes__label" for="f-roles">Case Manager</label>
+              </div>
```

Comment:
We could store these in an array and generate with `{{ range }}` rather than typing them all out

Reply:
meh, I've already done the typing now